### PR TITLE
feat/rr-functionality

### PR DESCRIPTION
**Pull request by joshenlim**
giphy

### CODE REVIEW

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx [hunk 1]
```diff
@@ -247,35 +251,44 @@ const MapView = ({
...
                             <DropdownMenuItem className="gap-x-2">
                               <Link
-                                href={`/project/${ref}/settings/database?connectionString=${database.id}`}
+                                href={`/project/${ref}/settings/database?connectionString=${database.identifier}`}
                               >
                                 View connection string
                               </Link>
                             </DropdownMenuItem>
-                            <DropdownMenuItem
+                            {/* <DropdownMenuItem
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx:291]: Intentionally commented out?

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceNode.tsx [hunk 2]
```diff
@@ -150,12 +161,12 @@ export const ReplicaNode = ({ data }: NodeProps<ReplicaNodeData>) => {
                   View connection string
                 </Link>
               </DropdownMenuItem>
-              <DropdownMenuItem className="gap-x-2" onClick={() => onSelectRestartReplica()}>
+              {/* <DropdownMenuItem className="gap-x-2" onClick={() => onSelectRestartReplica()}>
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceNode.tsx:164]: Intentionally commented out?

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx [hunk 3]
```diff
@@ -145,43 +197,18 @@ const InstanceConfigurationUI = () => {
...
-                replicas in this region
-              </li>
-            </ul>
-          </div>
-        </Modal.Content>
-      </ConfirmationModal>
+      /> */}
 
-      <ConfirmationModal
+      {/* <ConfirmationModal
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx:211]: Intentionally commented out?

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx [hunk 4]
```diff
@@ -145,43 +197,18 @@ const InstanceConfigurationUI = () => {
         }}
       />
 
-      <ResizeReplicaPanel
+      <DropReplicaConfirmationModal
+        selectedReplica={selectedReplicaToDrop}
+        onCancel={() => setSelectedReplicaToDrop(undefined)}
+      />
+
+      {/* <ResizeReplicaPanel
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx:205]: Intentionally commented out?

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx [hunk 5]
```diff
@@ -98,9 +183,9 @@ const DeployNewReplicaPanel = ({
           ))}
         </Listbox>
 
-        <p className="text-xs text-foreground-light">
+        {/* <p className="text-xs text-foreground-light">
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx:186]: Intentionally commented out?

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureActivity.tsx [hunk 6]
```diff
@@ -187,12 +188,12 @@ const InfrastructureActivity = () => {
       </ScaffoldContainer>
       <ScaffoldContainer className="sticky top-0 py-6 border-b bg-background z-10">
         <div className="flex items-center gap-x-4">
-          {readReplicasEnabled && (
+          {/* {readReplicasEnabled && (
```
Comment (alaister) [inline apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureActivity.tsx:191]: Intentionally commented out?

**Review by alaister — APPROVED** [review]
LGTM!

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceNode.tsx (hunk 2 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceNode.tsx:164]: https://github.com/supabase/supabase/pull/19466#discussion_r1419938468

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx (hunk 4 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx:207]: https://github.com/supabase/supabase/pull/19466#discussion_r1419938468

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx (hunk 5 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx:189]: https://github.com/supabase/supabase/pull/19466#discussion_r1419938468

File: apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureActivity.tsx (hunk 6 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureActivity.tsx:191]: https://github.com/supabase/supabase/pull/19466#discussion_r1419938468

**Pull request merged**